mlir/test/Dialect/Mesh/canonicalization.mlir

```diff
@@ -0,0 +1,72 @@
+// RUN: mlir-opt --canonicalize %s | FileCheck %s
+
```

Comment:
Seems the test for `mesh.all_to_all` is missing.

Reply:
Added tests for all-to-all.